src/assets/styles/main.scss

```
@@ -1,4 +1,4 @@
-/*
+/*!
 * # SPARQLER custom styles
 * https://github.com/CarbonLDP/sparqler
```

Now that I see it, we must also change this to point to the `carbonldp-ts-docs-engine` repository.
```suggestion
 * https://github.com/CarbonLDP/carbonldp-ts-docs-engine
```